### DISCUSSION TITLE
基于知识库的人类行为分析模型

### DISCUSSION THREAD

**Discussion by AIzealotwu** [InternLM/Tutorial#283] [project]
受第三节课启发，想到以此作为大模型在生物领域的应用的案例，前景值得推敲，需要查找数据集或手动注释。
基于项目截止时间在2.4日，打算开发一个以解决病人行为监管的大模型，作为其在医学相关的领域的一个浅显的应用，目前项目招人，有意加入者在底下评论即可。

**Comment by lishuang70345** [comment]
想要参加组队一下，wx18862196994
